### PR TITLE
Created Telugu translation

### PR DESCRIPTION
**Pull request by vivekkoya**
Created Telugu translation for Dark Reader extension

### CODE REVIEW

**Review by Gusted — APPROVED** [review]
Thanks!

File: src/_locales/tel.config [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+సూర్యాస్తమయం వద్ద ప్రారంభించండి
+
+@longitude
+రేఖాంశం
+
+@latitude
+అక్షాంశం
+
+@page_protected
+ఈ పేజీ బ్రౌజర్ ద్వారా రక్షించబడింది
```
Comment (Gusted) [inline src/_locales/tel.config:46]: Please use two sentences for this translation.

File: src/_locales/tel.config [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+@page_in_dark_list
+ఈ పేజీ చీకటి థీమ్ జాబితాలో ఉంది
```
Comment (Gusted) [inline src/_locales/tel.config:49]: Please use two sentences for this translation.

File: src/_locales/tel.config [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+@sepia
+సెపియా
+
+@only_for
+దీని కోసం మాత్రమే
+
+@only_for_description
+ప్రస్తుత సైట్ కోసం మాత్రమే ఎంపికలను వర్తించండి
+
```
Comment (Gusted) [inline src/_locales/tel.config:89]:
```suggestion


```

File: src/_locales/tel.config [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+@not_invert_listed
+విలోమ జాబితా కాదు
+
+@add_site_to_list
+జాబితాకు సైట్‌ను జోడించండి
+
+@setup_add_site_hotkey
+సైట్‌ను జోడించడానికి హాట్‌కీని సెటప్ చేయండి
+
```
Comment (Gusted) [inline src/_locales/tel.config:106]:
```suggestion


```

File: src/_locales/tel.config (hunk 1 above)
Comment (vivekkoya) [reply src/_locales/tel.config:46]: @page_protected
ఈ పేజీ
బ్రౌజర్ ద్వారా రక్షించబడింది

File: src/_locales/tel.config (hunk 2 above)
Comment (vivekkoya) [reply src/_locales/tel.config:49]: @page_in_dark_list
ఈ పేజీ
చీకటి థీమ్ జాబితాలో ఉంది

File: src/_locales/tel.config (hunk 1 above)
Comment (vivekkoya) [reply src/_locales/tel.config:46]: I have split the translation into two lines like the translations in other languages. I have done this for @page_protected and @page_in_dark_list.

Comment (vivekkoya) [reply src/_locales/tel.config:46]: Thank you for the feedback

File: src/_locales/tel.config [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+డార్క్ రీడర్ ప్రకటనలను చూపించదు మరియు వినియోగదారుల డేటాను ఎక్కడా పంపదు. ఇది పూర్తిగా ఓపెన్-సోర్స్: https://github.com/darkreader/darkreader
+
+మీరు దీన్ని ఇన్‌స్టాల్ చేయడానికి ముందు, ఇలాంటి పొడిగింపులను నిలిపివేయండి. చూడటం ఆనందించండి!
+
+
+
+
+
+
+
```
Comment (Gusted) [inline src/_locales/tel.config:190]:
```suggestion
```

**Pull request merged**